projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -184,6 +198,13 @@ class_exists( 'Jetpack_AMP_Support' )
 			// Override Notification module to include RTL styles.
 			add_action( 'a8c_wpcom_masterbar_enqueue_rtl_notification_styles', '__return_true' );
 		}
+
+		// Hides and replaces the language dropdown for the current user, on Atomic.
+		if ( jetpack_is_atomic_site() &&
+			defined( 'IS_PROFILE_PAGE' ) && IS_PROFILE_PAGE ) {
+			add_action( 'user_edit_form_tag', array( $this, 'hide_language_dropdown' ) );
+			add_action( 'profile_personal_options', array( $this, 'replace_language_dropdown' ) );
```

Comment:
Changing `personal_options` to `profile_personal_options` might have surfaced another issue, because I see the locale row under the "Post by Email" settings now:

<img width="972" alt="Screen Shot 2021-05-13 at 11 01 12" src="https://user-images.githubusercontent.com/1233880/118103945-c8eab700-b3da-11eb-8731-508146057184.png">

Reply:
Thanks for catching that, I added a priority to bump it up in https://github.com/Automattic/jetpack/pull/19525/commits/c46b2c916351d87c4a623869a97f474bb3d0f56f

<img width="536" alt="Screen Shot 2021-05-13 at 11 42 08 AM" src="https://user-images.githubusercontent.com/1689238/118150302-65fc1e80-b3e0-11eb-85d1-9e7f4c9aef84.png">

Reply:
Awesome! Do you think we can get rid of that top margin so there is the same separation across all the personal options?

<img width="993" alt="Screen Shot 2021-05-14 at 12 23 11" src="https://user-images.githubusercontent.com/1233880/118258154-d2921e80-b4af-11eb-9121-346b9db77363.png">

Otherwise, it feels like the language is a bit off:

<img width="889" alt="Screen Shot 2021-05-14 at 12 28 20" src="https://user-images.githubusercontent.com/1233880/118258318-fead9f80-b4af-11eb-81b5-9ad3f03582cd.png">

Reply:
Yikes yeah. I think that's happening because `profile_personal_options` is outside of the personal options table. It's in its own table now, with a `form-table` class that adds padding. Should we move it back into `personal_options`, or use custom CSS / classes to reduce the space?

Reply:
Either way looks good to me. Moving it back to `personal_options` seems simpler and less hacky (since we want to use the personal options table), but keeping it in `profile_personal_options` could be seen as more readable/explicit (it's clear that changes are only applied to the profile page). I'm fine with whatever option you choose.

Reply:
Let's go back to `personal_options`. I made the change here: https://github.com/Automattic/jetpack/pull/19525/commits/8dc3b1742e7631c887f74cfb8850dad4150cba37